src/web/app/components/status-message/status-message.component.spec.ts

```diff
@@ -19,7 +20,50 @@ describe('StatusMessageComponent', () => {
     fixture.detectChanges();
   });
 
+  const messages: StatusMessage[] = [
+    { message: 'a', color: 'black' },
+    { message: 'b', color: 'red' },
+    { message: 'c', color: 'green' },
+  ];
+
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with default view', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should show all three messages', () => {
+    component.messages = messages;
+    fixture.detectChanges();
+    const statusMessages: HTMLElement[] = fixture.nativeElement.querySelectorAll('div.alert div');
+    expect(statusMessages.length).toEqual(3);
+    expect(statusMessages[0].innerHTML).toEqual('a');
+    expect(statusMessages[1].innerHTML).toEqual('b');
+    expect(statusMessages[2].innerHTML).toEqual('c');
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should delete a message after clicking button', () => {
```

Comment:
Try to separate/grouping the `setup`, `execution` and `verification` parts by inserting emtpy lines in the test?